src/backend/main/py.main/caliopen_main/parsers/vcard.py

```diff
@@ -193,6 +193,18 @@ def parse_vcard(vcard):
                                 ke.type = j
                     ke.name = ('key{}{}'.format(ke.type,ke.size))
                     new_contact.public_keys.append(ke)
+
+        elif v == 'tel':
+            for tel in vcard.contents['tel']:
+                phone = NewPhone()
+                phone.is_primary = False
+                try:
+                    func = PhoneNumberType.validate_phone
+                    phone.number = func(PhoneNumberType(),tel.value)
+                    new_contact.phones.append(phone)
+                except:
+                    pass
```

Comment:
does it mean that a phone is not well formed it will be drop ?

for e.g: `01 02 03 04 05` is not "valid" cf. #345 , so it will not be saved ?
Is there any other way ?

may be a normalizer or whatever ? Or at least save in custom fields ?